### PR TITLE
chore: remove unused package

### PR DESCRIPTION
**Pull request by eseidel**
Much more to remove here, but this is a start.

### CODE REVIEW

**Comment by eseidel** [comment]
I don't understand the "patch" version change.

**Comment by eseidel** [comment]
In cargo.lock

File: Cargo.lock [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ dependencies = [
 
 [[package]]
 name = "patch"
-version = "0.1.0"
+version = "0.2.1"
```
Comment (eseidel) [inline Cargo.lock:1021]: I don't understand this change.

Comment (eseidel) [reply Cargo.lock:1021]: But our "patch" library is version = "0.2.1" 🤷

**Comment by bryanoltman** [comment]
Patch is version 0.2.1 (see [here](https://github.com/shorebirdtech/updater/blob/main/patch/Cargo.toml#L3)). I believe it's just updating to reflect that.

**Review by bryanoltman — APPROVED** [review]

**Pull request merged**